test/routes/src/server/routes/t_github.js

```diff
@@ -640,12 +641,21 @@ const hasStatus = (expected) => {
   };
 };
 
-const hasMessage = (expected) => {
+const hasPayloadMessage = (expected) => {
```

Comment:
`hasPayloadMessage` is checking for a code property inside `action.payload` whereas `hasBodyMessage` is checking for a code property at the top level of `action`. This is part of the gradual removal of the `payload` property from API responses, and when it has been removed across the board `hasPayloadMessage` can be removed.

I think it's worth doing this work in stages, but open to discussion about whether a different approach should be taken.